app/src/main/java/com/example/weathertimeapp/mvp/contracts/WeatherTimeContracts.kt

```diff
@@ -0,0 +1,17 @@
+package com.example.weathertimeapp.mvp.contracts
+
+import com.example.weathertimeapp.mvp.entities.City
+
+interface WeatherTimeContracts {
+    interface Model {
+        fun createListOfCities(listOfCities: MutableList<City>): MutableList<String>
```

Comment:
can this be List instead of MutableList?

Reply:
yes it can, done